cohortextractor/emis_backend.py

```diff
@@ -20,30 +20,7 @@
 safe_punctation = r" _.-+/()"
 SAFE_CHARS_RE = re.compile(f"^[a-zA-Z0-9{re.escape(safe_punctation)}]+$")
 
-
-# patient_all_orgs_v2 contains a handful of duplicate registration IDs.  This causes
-# problems: because of the way we build our final query with a series of N LEFT JOINs, a
-# registration ID appearing twice in the patient table can appear up to 2^N times.
-#
-# EMIS have not yet fixed this, so for now I have created a table called
-# patient_no_duplicates which removes these duplicate registration IDs.  The number of
-# duplicates is small enough that it doesn't matter that it causes us to lose a few
-# patients, and in any case there is no way to choose between patients with duplicate
-# registration IDs.
-#
-# WE MUST REMEMBER TO RECREATE THIS TABLE EACH TIME THE DATA IS REFRESHED!!
-#
-# CREATE TABLE patient_no_duplicates AS (
-#     SELECT *
-#     FROM patient_all_orgs_v2
-#     WHERE registration_id NOT IN (
-#         SELECT registration_id
-#         FROM patient_all_orgs_v2
-#         GROUP BY registration_id
-#         HAVING COUNT(*) > 1
-#     )
-# )
-PATIENT_TABLE = "patient_no_duplicates"
+PATIENT_TABLE = "patient_all_orgs_v2"
```

Comment:
I'm a bit confused by this switch.

Reply:
Have I explained this in https://github.com/opensafely-core/cohort-extractor/pull/506#issuecomment-806698656?

Reply:
You have. Thank you 🙂